### ISSUE TITLE
updating wavenet_params 

### ISSUE THREAD

**Issue by nakosung** [ibab/tensorflow-wavenet#227]
According to recent talk (https://www.youtube.com/watch?v=nsrSrYtKkT8)
(1,2,4,...1024) (1,2,4,...1024) (1,2,4,...1024) 30 layers

![image](https://cloud.githubusercontent.com/assets/2463571/23220316/ec155dfa-f964-11e6-8aaf-6478afc8421f.png)

**Comment by lmaxwell** [comment]
Did Heiga mention 30layers or ( 1,2,4,...,124) in the talk? 3 stacks of (1,2,4,...1024) is actually 33 layers.

**Comment by nakosung** [comment]
@lmaxwell Yes. He mentioned "1024" and "three-times". (https://youtu.be/nsrSrYtKkT8?t=24m41s)

**Comment by lmaxwell** [comment]
@nakosung thanks, I'll try this. Also from the image above, the residual channel is 512, it 's really big.

**Comment by nakosung** [comment]
@lmaxwell Yes. My TitanXP cannot afford such a big model.

**Comment by ucasyouzhao** [comment]
@nakosung IMO, the residual channel is 512, the dilation channel is 512 and the skip channel is 256.

**Comment by ucasyouzhao** [comment]
@nakosung From the image above, the residual channel is 512, the dilation channel is 512 and the skip channel is 256.  Is it right?

**Comment by nakosung** [comment]
@ucasyouzhao I think so. :)

**Comment by greaber** [comment]
Hey, I just looked at the video.  I think you guys are right about the residual, dilation, and skip channels.  But I think the dilation stack is 30 layers, (1,...,512,1,...,512,1,...,512).  This agrees with the Wavenet paper; on page 3 they actually show exactly this stack.  And it produces 30 layers, whereas going to 1024 would produce 33 as @lmaxwell points out.  He also says that stacking the thing three times lets it extract about 3,000 timesteps.  He says 1024 because that is about what the receptive field is.

By the way, he says something like, "At this moment you were using a 10 stack, so 1024," suggesting that perhaps they have moved on to using different parameters.

**Comment by nakosung** [comment]
I updated json. :)

**Comment by rvijay1234** [comment]
The receptive field as @dannybtran states is 192ms. Howver, in the paper they have mentioned that for the 'Multi-speaker Speech Generation' experiments they have used a model with a receptive field of about 300ms.
And it was 240ms in case of 'Text-to-speech'. Since this talk is based on text-to-speech, it would be wise to have 2 separate models.
